src/index.html

```diff
@@ -17,11 +17,434 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a href="#">
+            <img
+              class="header__logo-png"
+              src="./images/logo.png"
+              alt="Moyo logo"
+            />
+          </a>
+        </div>
+        <nav class="navigation">
+          <ul class="navigation__container">
+            <li class="navigation__container-list">
+              <a
+                href="#"
+                class="navigation__container-list-link is-active"
```

Comment:
Fix classes everywhere
```suggestion
        <nav class="nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a
                href="#"
                class="nav__link is-active"
```